CI.md

```diff
@@ -0,0 +1,14 @@
+# Buildkite
+
+These are the instructions for generating nix expressions for buildkite CI for log-classifier.
+
+```
+$ cd $LOG_CLASSIFIER_ROOT
+$ cabal2nix . > default.nix
```

Comment:
I don't think that `cabal2nix` will capture all of the contents of the Stack project. Ideally, we should use `stack2nix` here, it might explain why you needed to add `Semigroup` - because it picked up the latest version of GHC, disregarding the version we put out in `stack.yaml`?
Anyway, good enough for now, you can just create another ticket for this specific issue.

Reply:
I tried to use stack2nix, but got errors like:
```error: Package ‘Win32-2.5.4.1’ in /home/coder/Nextcloud/IOHK/log-classifier/default.nix:1943 is not supported on ‘x86_64-unknown-linux-gnu’, refusing to evaluate.```

I spent some time trying to fix this but I couldn't crack it. Any idea how to fix this?

Reply:
No, but re-read my comment above and my comments in the channel.